engine/src/main/java/nl/inl/blacklab/search/results/Hits.java

```diff
@@ -478,18 +486,18 @@ public static Hits mutableEmptyList(QueryInfo queryInfo) {
      * The number of hits we've seen and counted so far. May be more than the number
      * of hits we've retrieved if that exceeds maxHitsToRetrieve.
      */
-    protected int hitsCounted = 0;
+    private int hitsCounted = 0;
```

Comment:
@severian I am not sure if I am following 100% why we made some of this variables private, Any insights here will be appreciated.

Reply:
It's because they need to be overridden in `HitsFromQuery`, so they can be read from the atomic context. I had forgot to override some in my last PR. I added them in https://github.com/lexionai/BlackLab/pull/19